### PR TITLE
Fix bad wasm remote load test flaky.

### LINKED ISSUE #30593: TestBadWasmRemoteLoad test flake
https://prow.istio.io/view/gs/istio-prow/logs/integ-telemetry-mc-k8s-tests_istio_postsubmit/1356671068818903040

### PR DESCRIPTION
**Pull request by bianpengyuan**
fixes #30593

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: tests/integration/telemetry/stats/prometheus/wasm/bad_wasm_filter_test.go [hunk 1]
```diff
@@ -39,7 +40,14 @@ func TestBadWasmRemoteLoad(t *testing.T) {
 		Run(func(ctx framework.TestContext) {
 			// Test bad wasm remote load in only one cluster.
 			// There is no need to repeat the same testing logic in several different clusters.
-			cltInstance := common.GetClientInstances()[0]
+			var cltInstance echo.Instance
+			for _, i := range common.GetClientInstances() {
```
Comment (howardjohn) [inline tests/integration/telemetry/stats/prometheus/wasm/bad_wasm_filter_test.go:44]: could probably be: `common.GetClientInstances().GetOrFail(ctx, echo.InCluster(ctx.Clusters().Default()))`

**Review by howardjohn — APPROVED** [review]

**Comment by bianpengyuan** [comment]
@istio/wg-policies-and-telemetry-maintainers ptal, thanks!

**Review by mandarjog — APPROVED** [review]
Lgtm

**Pull request merged**